tests/stress_tests/README.md

```diff
@@ -32,27 +32,12 @@ one at a time to mitigate memory statistics pollution. You can use
 
 ### Building Tests
 
-Stress tests should be built in 2 steps.
-
-1. Build `openvino`
-
-Build `openvino` as usual but with `-DENABLE_TESTS=ON`.
-
-2. Build `stress_tests`
-
-Stress tests depend from the Inference Engine Developer Package located in the
-`openvino` build directory.
-
-In the command line snippet bellow, it is assumed that the Inference Engine
-Developer Package CMake module can be found in the directory `build` under
-`openvino` repository root.
-
+To build the tests, you need to have OpenVINO™ installed or build from source.
+Before build the tests, open a terminal, set OpenVINO™ environment, and after that
+run the commands below:
 ``` bash
-(
-export OPENVINO_BUILD_DIR=$(git rev-parse --show-toplevel)/build
-mkdir -p build && cd build && \
-cmake -DInferenceEngineDeveloperPackage_DIR=$OPENVINO_BUILD_DIR .. && make -j$(nproc) \
-)
+mkdir build && cd build
+cmake .. && make
```

Comment:
When building with OpenVINO from github how to set up env vars so the package can be found?

Reply:
Need to install it and call setupvars. Do you mean it should be documented here too?

Reply:
README changes are moved to https://github.com/openvinotoolkit/openvino/pull/6786